### DISCUSSION TITLE
autoref style cross-referencing for PDF engine

### DISCUSSION THREAD

**Discussion by halidaee** [quarto-dev/quarto-cli#2443] [Feature Requests]
Currently, Quarto's cross-referencing uses TeX defaults, so when referencing "Figure 1", only the text for "1" is hyperlinked to the relevant figure. (This is only for PDFs. All of "Figure 1" is linked in HTML.) It would be nice to add the functionality to make all the text of "Figure 1" link to the figure, and do so analogously for other objects. This is available in TeX using the hyperref package in latex using the autoref macro.

**Comment by mcanouil** [comment]
Related to https://github.com/quarto-dev/quarto-cli/issues/4944
